janusgraph-test/src/test/java/org/janusgraph/graphdb/management/ConfigurationManagementGraphTest.java

```diff
@@ -64,4 +72,22 @@ public void shouldReindexIfPropertyKeyExists() throws Exception {
         assertEquals(ENABLED, index.getIndexStatus(propertyKey));
         management.commit();
     }
+
+    @Test
+    public void shouldCloseAllTxsIfIndexExists() {
+        final JanusGraphManager gm = new JanusGraphManager(new Settings());
```

Comment:
(nitpick) You aren't using `gm` anywhere. So, you just remove this line.

Reply:
Ok